README.md

```diff
@@ -102,8 +102,11 @@ More Information  [llvm-cov](https://github.com/llvm-mirror/clang/blob/master/do
 ##### 3. Enable "Publish Coverage Report" publisher in the Post-build Actions.
 ##### 4. Add your coverage tool adapter and specify reports path.
 ![alt text](./images/config-add-adapter.png "Add coverage adapter")
-##### 5. (Optional) Specify Thresholds of each metrics in global or adapter level.
-##### 6. (Optional) Specify Source code storing level to enable source code navigation.
+##### 5. (Optional) Let discover the reference build to get delta coverage compared to the reference build. 
+Use the `discoverGitReferenceBuild()` of the [forensics-api](https://github.com/jenkinsci/forensics-api-plugin) plugin.
+You can use it directly and don't have to install the dependency, as it's a mandatory dependency!
```

Comment:
It is a little bit more complex: the forensics API ist mandatory, but does not provide the `discoverGitReferenceBuild` step. This step is part of the git-forensics plugin (the only implementation of the API plugin so far). But using the forensics API  helps anyway since you can use it in freestyle jobs as well to compute a reference build. 

So I suggest to shorten as (if someone wants to read the details, then the documentation of the forensics API should be enough). 
```suggestion
##### 5. (Optional) Use the [forensics-api](https://github.com/jenkinsci/forensics-api-plugin) plugin to discover the reference build that is used to compute a delta report of the coverage results. 
```